static/app/components/feedback/feedbackItem/feedbackShortId.tsx

```diff
@@ -64,6 +64,11 @@ export default function FeedbackShortId({className, feedbackItem, style}: Props)
     text: feedbackItem.shortId,
   });
 
+  const {onClick: handleCopyMarkdown} = useCopyToClipboard({
+    text: `[${feedbackItem.shortId}](${feedbackUrl})`,
+    successMessage: t('Copied Markdown Link to clipboard'),
```

Comment:
```suggestion
    successMessage: t('Copied Markdown Feedback Link to clipboard'),
```
to match the issues one

Reply:
<img width="337" alt="SCR-20241015-jmwh" src="https://github.com/user-attachments/assets/6bb7e8b7-2435-40ef-9226-6cc186fa51b4">